<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PublisherForm</class>
 <widget class="QWidget" name="PublisherForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="descriptionLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="critiqueLabel">
     <property name="text">
      <string>Critique:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="critiqueLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="logoLabel">
     <property name="text">
      <string>Logo:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="logoWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="logoLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="logoPushButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="themesLabel">
     <property name="text">
      <string>Themes:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QListWidget" name="themesListWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="cancelPushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="savePushButton">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>descriptionLineEdit</tabstop>
  <tabstop>critiqueLineEdit</tabstop>
  <tabstop>logoLineEdit</tabstop>
  <tabstop>logoPushButton</tabstop>
  <tabstop>themesListWidget</tabstop>
  <tabstop>cancelPushButton</tabstop>
  <tabstop>savePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
